test/e2e/tests/network/switch-network.spec.ts

```diff
@@ -0,0 +1,66 @@
+import { Suite } from 'mocha';
+import { Driver } from '../../webdriver/driver';
+import { withFixtures, defaultGanacheOptions } from '../../helpers';
+import FixtureBuilder from '../../fixture-builder';
+import { Ganache } from '../../seeder/ganache';
+import { loginWithBalanceValidation } from '../../page-objects/flows/login.flow';
+import HomePage from '../../page-objects/pages/homepage';
+import ModalConfirmation from '../../page-objects/pages/dialog/network-switch-modal-confirmation';
+import HeaderNavbar from '../../page-objects/pages/header-navbar';
+import SelectNetwork from '../../page-objects/pages/dialog/select-network';
+
+describe('Switch network - ', function (this: Suite) {
+  it('multiple networks', async function () {
```

Comment:
nit: I think the test description could be more explicit on what the test does. Given you opt for having all in the same test, something like:  
```suggestion
  it('Switch networks to existing and new networks', async function () {
```

Reply:
@seaona Thanks! Yeah, I agree this is much better. I've pushed up the changes.